### PR TITLE
Antihol now makes surgeries slower

### PR DESCRIPTION
**Pull request by Exester509**
## About The Pull Request

Splashing antihol on someone will now cause surgeries to go a bit slower. This should completely counteract ethanol's speed buff.

## Why It's Good For The Game

Someone suggested this on the discord and I found it a funny idea, adds a bit more flavor to antihol as the opposite of ethanol.

## Changelog

:cl:
add: Splashing antihol on a patient before surgery will make it to go slower.
/:cl:

### CODE REVIEW

**Comment by Exester509** [comment]
I may have fucked up

**Comment by AtomTheProphet** [comment]
Doesn't ethanol only accelerate surgeries because it sterilizes surfaces? Also, isn't antihol the opposite of prohol, and vice versa?

I'm not against this, I just think it's a bit odd.

**Comment by atlasle** [comment]
> Doesn't ethanol only accelerate surgeries because it sterilizes surfaces? Also, isn't antihol the opposite of prohol, and vice versa?
> 
Now you've got me thinking, what if spreading virus food on someone makes surgery harder

**Comment by the-orange-cow** [comment]
antihol smoke machine in medbay, gogogo!

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 1]
```diff
@@ -1110,6 +1110,14 @@
 	..()
 	. = TRUE
 
+/datum/reagent/medicine/antihol/expose_mob(mob/living/carbon/exposed_carbon, methods=TOUCH, reac_volume)
+	. = ..()
+	if(!(methods & (TOUCH|VAPOR|PATCH)))
+		return
+
+	for(var/datum/surgery/surgery as anything in exposed_carbon.surgeries)
+		surgery.speed_modifier -= 0.1
```
Comment (Ghommie) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm:1119]:
```suggestion
		surgery.speed_modifier = max(surgery.speed_modifier  - 0.1, -0.9)
```
a speed modifier of -1 will lead to division by zero, and any lower than that, to a negative time.

**Pull request merged**